src/commands/generate/index.ts

```diff
@@ -72,7 +72,9 @@ export default class Generator extends Command {
 
       await this.postProcess(generalOptions);
 
-      this.log('The infrastructure has been generated!');
+      this.log(
+        `The infrastructure code was generated at "${generalOptions.projectName}"`
```

Comment:
Can we use `/`` instead?
```suggestion
        `The infrastructure code was generated at \`${generalOptions.projectName}\``
```
If you agree, please change the test as well

Reply:
it's better. updated in 9155bce